cmd/vicadmin/server.go

```
@@ -166,36 +352,51 @@ func (s *server) stop() error {
 	return nil
 }
 
+// logout handler expires the user's session cookie by setting its creation time to the beginning of time
+func (s *server) logoutHandler(res http.ResponseWriter, req *http.Request) {
+	websession, err := s.uss.cookies.Get(req, sessionCookieKey)
```

yeap you're right, we can ignore this error.